db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+ book1 = Book.create(title: "Last of the mohicans")
+ book2 = Book.create(title: "First of the Mohicans")
+ book3 = Book.create(title: "Scarlett Letter")
+ book4 = Book.create(title: "Left Hand of Darkness")
+ book5 = Book.create(title: "Right Hand of Darkness")
+ book6 = Book.create(title: "Mary Poppinns")
+
+
+ user1 = User.create(name: "Tom")
+ user2 = User.create(name: "Jen")
+ user3 = User.create(name: "Sara")
+ user4 = User.create(name: "Sam")
+ user5 = User.create(name: "Bob")
+ user6 = User.create(name: "Rachel")
+
+
+review1 = user1.reviews.create(body: 'it was awful', rating: '' )
```

Comment:
In a timed assessment like this I'd probably skip a seed file. You won't need it if you're using TDD. The setup will happen in each test instead.